core/pkg/service/sync/sync_metrics.go

```diff
@@ -30,7 +30,7 @@ func (s *Server) captureMetrics() error {
 	}
 
 	_, err = meter.RegisterCallback(func(_ context.Context, o api.Observer) error {
-		o.ObserveInt64(syncGuage, s.handler.syncStore.GetActiveSubscriptionsInt64())
+		o.ObserveInt64(syncGuage, s.oldHandler.syncStore.GetActiveSubscriptionsInt64())
```

Comment:
I think we'll want to add the sums for both handlers here, or otherwise handle the fact that functionally we want to aggregate metrics from both endpoints.

Reply:
Ah that makes sense... and I see you're back to using `s.handler` which will work going forward after we remove `oldHandler`. :+1: